crossdock/docker-compose.yml

```diff
@@ -33,27 +33,56 @@ services:
             - REPORT=compact
     go:
         build: .
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     java:
         image: jaegertracing/xdock-java
         ports:
             - "8080-8082"
-#       Udp sender needs to know agent's address
+        links:
+            - "jaeger:jaeger-agent"
         depends_on:
-            - jaeger-agent
+            # Udp sender needs to know agent's address
+            - jaeger
 
     python:
         image: jaegertracing/xdock-py
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080:8082"
 
     test_driver:
         image: jaegertracing/test-driver
+        links:
+            - "jaeger:jaeger-agent"
+            - "jaeger:jaeger-collector"
+            - "jaeger:jaeger-query"
         depends_on:
-            - jaeger-query
-            - jaeger-collector
-            - jaeger-agent
+            - jaeger
         ports:
             - "8080"
+        environment:
+            - JAEGER_QUERY_HC_HOST_PORT=jaeger-query:14269
+
+    jaeger:
+        image: jaegertracing/all-in-one
+        ports:
+            - "14269"
```

Comment:
Do we need to expose all these ports? Exposing a smaller set is more explanatory on what us being used.